app/models/comment.rb

```diff
@@ -77,13 +77,25 @@ def mentioned_users
     User.find_all_by_username(usernames.map {|m| m[1] }).uniq
   end
 
+  def mentioned_tag_followers
```

Comment:
Ha, i see, grammatically that's quite difficult, huh? What about `followers_of_mentioned_tags`? Thanks!